packages/core/src/mastra/index.ts

```diff
@@ -532,6 +532,35 @@ do:
     return this.#serverMiddleware;
   }
 
+  public setServerMiddleware(serverMiddleware: Middleware | Middleware[]) {
```

Comment:
Are we planning on reading the middleware and then setting back a full list of middleware?